src/python/pyqgl2/lang.py

```diff
@@ -23,8 +21,8 @@ class QGL2(object):
 
     CLASSICAL = 'classical'
     PULSE = 'pulse'
-    QBIT = 'qbit'
-    QBIT_LIST = 'qbit_list'
+    QBIT = 'qreg'
+    QBIT_LIST = 'qreg_list'
```

Comment:
One open question is whether we bother to rename these internal symbols: `QGL2.QBIT` and `QGL2.QBIT_LIST`.